tests/test_volume.py

```diff
@@ -230,25 +230,24 @@ def testRotateBroadcastUnicast(self):
         for i in range(self.n):
             self.assertTrue(np.allclose(vols_broadcast[i], vols_unicast[i]))
 
-    def testCnSymmetricVolume(self):
+    @parameterized.expand([(64,), (65,)])
+    def testCnSymmetricVolume(self, L):
```

Comment:
Why are these test not in `test_synthetic_volume`?

Reply:
moved